src/i18n/en-US/systemProfile.ts

```diff
@@ -75,6 +75,57 @@ const systemProfile = {
       noATODates:
         'There is no information about previous tests or assessments available for this system'
     },
+    fundingAndBudget: {
+      header: 'Funding and Budget',
+      actualFYCost: 'Actual FY system cost',
+      budgetedFYCost: 'Budgeted FY system cost',
+      investmentNumber: 'Investment number',
+      requisitionNumber: 'Requisition number',
+      tagHeader1: 'Funding Sources',
+      budgetID: 'Budget ID: '
+    },
+    teamAndContract: {
+      header: {
+        teamAndContract: 'Team and Contract',
+        contractInformation: 'Contract Information',
+        pointsOfContact: 'Points of Contact'
+      },
+      federalFullTimeEmployees: 'Federal Full Time Employees',
+      contractorFullTimeEmployees: 'Contractor Full Time Employees',
+      vendors: 'Vendors',
+      contractAwardDate: 'Contract Award Date',
+      periodOfPerformance: 'Period of performance',
+      startDate: 'Start Date',
+      endDate: 'End Date',
+      contractNumber: 'Contract number',
+      technologyFunctions: 'Technology functions',
+      assetsOrServices: 'Assets or services',
+      pointOfContact: 'Point of contact',
+      sendAnEmail: 'Send an email'
+    },
+    toolsAndSoftware: {
+      header: 'Tools and Software',
+      productType: 'Product Type',
+      softwareVersion: 'Software version',
+      softwareEdition: 'Software edition'
+    },
+    systemData: {
+      header: 'System Data',
+      recordCategories: 'Records management categories',
+      beneficiariesAddress: 'Benfeficiary addresses',
```

Comment:
Typo here in the spelling of "Beneficiary".

Reply:
Fixed with https://github.com/CMSgov/easi-app/pull/1477/commits/8b42dad54bc37832daeda5ca461bdd3419582a2b.